settings/badge_store_settings.py

```diff
@@ -0,0 +1,9 @@
+from stack_nav import *
+from database import *
```

Comment:
does this need license or dependencies set?

Reply:
no, only main.py needs dependencies for apps (I should document that)